packages/strapi-design-system/src/Divider/Divider.js

```diff
@@ -6,6 +6,8 @@ import { Box } from '../Box';
 const DividerWrapper = styled(Box)`
   height: 1px;
   border: none;
+  /* If contained in a Flex parent we want to prevent the Divider to shink */
+  flex-shrink: 0;
```

Comment:
without `flex-shrink: 0` here the `Divider` ends up being shrinked when there's not enough space (e.g. when the `MainNav` enters scrollable display)
is there another solution to fix that? 
<img width="222" alt="image" src="https://user-images.githubusercontent.com/71838159/212296817-05560b70-c9ab-406f-81fa-69e1348fac80.png">

(cc @joshuaellis @gu-stav)

Reply:
I think that's fine, alternatively (although maybe not possible) you could have wrapped the top bit with the divider in this circumstance.

Reply:
it would be possible but then you have to think about adding a `Box` when building your `MainNav` because you can choose to add a `Divider` or not, it's not integrated with the top "Strapi Dashboard" title:
<img width="484" alt="image" src="https://user-images.githubusercontent.com/71838159/212371321-0db705ba-9bc1-41f1-a349-5e3bd96f2d27.png">

Reply:
Yeah I thought so, it's fine as is 👍🏼